packages/hint-validate-summary/package.json

```diff
@@ -0,0 +1,79 @@
+{
+    "ava": {
+        "failFast": false,
+        "files": [
+            "dist/tests/**/*.js"
+        ],
+        "timeout": "1m"
+    },
+    "description": "testing",
+    "devDependencies": {
+        "@hint/parser-css": "^3.0.2",
+        "@hint/utils-tests-helpers": "^5.0.0",
+        "@types/debug": "^4.1.4",
+        "@types/node": "^12.0.7",
+        "@typescript-eslint/eslint-plugin": "^1.10.2",
+        "@typescript-eslint/parser": "^1.10.2",
+        "ava": "^1.4.1",
+        "cpx": "^1.5.0",
+        "eslint": "^5.15.1",
+        "eslint-plugin-markdown": "^1.0.0",
+        "hint": "^5.0.2",
+        "npm-link-check": "^3.0.0",
+        "npm-run-all": "^4.1.5",
+        "nyc": "^14.1.0",
+        "postcss": "^7.0.17",
+        "rimraf": "^2.6.3",
+        "typescript": "^3.5.1"
+    },
+    "dependencies": {
+        "@hint/utils": "^2.1.0",
+        "postcss": "^7.0.17",
+        "postcss-selector-parser": "^6.0.2"
+    },
+    "engines": {
+        "node": ">=8.0.0"
+    },
+    "files": [
+        "dist/src"
+    ],
+    "homepage": "https://webhint.io/",
+    "keywords": [
+        "hint",
+        "hint",
+        "test",
+        "test-hint"
+    ],
+    "license": "Apache-2.0",
+    "main": "dist/src/hint.js",
+    "name": "@hint/hint-test",
+    "nyc": {
+        "extends": "../../.nycrc"
+    },
+    "peerDependencies": {
+        "hint": "^5.0.2",
+        "@hint/parser-css": "^3.0.2"
```

Comment:
Minor, but this should be listed before `"hint"` to be consistent with the sorting we do in other dependency sections.